adal/tests/Test.ADAL.NET.UIAutomation/AppFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+
+namespace Test.ADAL.NET.UIAutomation
+{
+    /// <summary>
+    /// Initializes the app object that represents the main gateway to interact with the app on the device
+    /// </summary>
+    public class AppFactory
+	{
+		public static IApp StartApp(Platform platform)
+		{
+            switch(platform)
+            {
+                case Platform.Android:
+                    return ConfigureApp.Android.InstalledApp("com.Microsoft.XFormsDroid.ADAL").StartApp();
+                case Platform.iOS:
+                    return ConfigureApp.iOS.StartApp();
+                default:
+                    return ConfigureApp.Android.InstalledApp("com.Microsoft.XFormsDroid.ADAL").StartApp();
```

Comment:
> return ConfigureApp.Android.InstalledApp [](start = 19, length = 41)

IMO: this should throw an exception saying platform not supported

Reply:
@henrik-me I agree